balancer/settings.py

```diff
@@ -11,6 +11,9 @@
 """
 
 from pathlib import Path
+from dotenv import load_dotenv
```

Comment:
Would you be against passing these env variables into the container, rather than using dotenv? I'm asking in order to avoid needing to copy env files into our backend related containers